assets/queries/openAPI/parameter_object_incorrect_ref/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "d40f27e6-15fb-4b56-90f8-fc0ff0291c51",
+  "queryName": "Parameter Object With Incorrect Ref",
```

Comment:
The folder name is slightly different from the 'queryName' in the metadata file

Reply:
I think it's ok